packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -300,6 +300,11 @@ export class CdpAutomation {
           return this.getCookie(data)
         })
 
+      case 'add:cookies':
+        setCookie = data.map((cookie) => normalizeSetCookieProps(cookie)) as Protocol.Network.SetCookieRequest[]
```

Comment:
Wonder if there is additional benefit to specifying the setCookie type to show it might be a single SetCookieRequest or an array of them on line 278, and then the consuming code just casts it, though I don't think this has a lot of benefit from a typings perspective.

```
let setCookie: Protocol.Network.SetCookieRequest | Protocol.Network.SetCookieRequest[]
```